File: Fibonacci/Fibonacci.cpp
// C++ program to print Fibonacci series upto n terms
#include <iostream>
using namespace std;

int main(){

    int n;
    cout << "Enter n: ";
    cin >> n;

    int a = 0, b = 1, c = 0;

    for(int i = 0; i < n; i++){
        cout << c << " ";

        a = b;
        b = c;
        c = a+b;
    }

    return 0;

}

File: Fibonacci/Fibonacci.py
# Python Program for printing Fibonacci Numbers upto N

def printFibonacciNumbers(n):
 
    a = 0
    b = 1
    if (n < 1):
        return
    for x in range(n):
        print(a, end=" ")
        a, b = b, a+b
 
 
# Driver code
if __name__ == "__main__":
    n = int(input("Enter N: "))
    printFibonacciNumbers(n)
